.github/workflows/build.yml

```
@@ -798,6 +810,15 @@ jobs:
           # pysoundfile
           ln -sf "${{ env.PYTHON_SITE_PACKAGES_DIR }}/_soundfile_data" artifact/
 
+      - name: Code signing
+        if: github.event.inputs.code_signing
```

`github.event.inputs.code_signing` は文字列、とのことなので、以下の修正が必要と思いました！ https://github.com/Hiroshiba/check_code_signing/runs/7269556780?check_suite_focus=true で `github.event.inputs.code_signing` が `false` なのになぜ署名しようとしているのだろうと疑問に思ったので気づきました。

（基本的には `!==` 演算子を使うべきと思ったりしたのですが、他の箇所では基本的に `!=` を使っている感じだと思うので、そちらに合わせました……）

```suggestion
        if: github.event.inputs.code_signing != 'true'
```

---

修正が漏れていました！！ありがとうございます！！